tests/test_sentry.py

```diff
@@ -84,6 +85,40 @@ def test_pywarnings(pywarnlogger):
     assert len(pywarnlogger.handlers) == 1
 
 
+def test_pywarnings_are_logged_once(pywarnlogger):
```

Comment:
Perhaps just one tiny improvement would be to factor out part of it as a fixture:

```
# conftest.py
@pytest.fixture()
def sentry_emit()
    from raven.handlers.logging import SentryHandler
    with patch.object(SentryHandler, 'emit') as sentry_emit:
        yield sentry_emit

#tests_....py
def test_pywarnings_are_logged_once(pywarnlogger, sentry_emit): 
    from invenio_logging.sentry import InvenioLoggingSentry
    # ...
```

The tests in the logging module do not make much use of fixtures though

Reply:
Thank you! Yes, that's much cleaner. I'll change that.